src/braket/aws/aws_device.py

```diff
@@ -79,8 +79,7 @@ class AwsDevice(Device):
     }
 
     def __init__(self, arn: str, aws_session: Optional[AwsSession] = None):
-        """
-        Args:
+        """Args:
```

Comment:
This indentation looks subjectively worse to me, but if it's consistent then I don't mind

Reply:
Indentation is based on https://peps.python.org/pep-0257/ and https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html. We can change back using https://docs.astral.sh/ruff/rules/multi-line-summary-second-line/ which is a counterpart to [D212](https://docs.astral.sh/ruff/rules/multi-line-summary-first-line) - the rule driving this style.

Reply:
Nice receipts! I mean more specifically the case where we have "Args: " immediately after the triple quote; I don't see any examples showing this and it seems kinda funny. Maybe we should just add a summary line

Reply:
Yes good callout, added that in